test/parallel/test-buffer-arraybuffer.js

```diff
@@ -105,3 +105,17 @@ b.writeDoubleBE(11.11, 0, true);
     return true;
   });
 }
+
+{
+  // If byteoffset is not numeric, it defaults to 0.
+  const ab = new ArrayBuffer(10);
+  const buf = Buffer.from(ab, 'fhqwhgads');
+  assert.deepStrictEqual(buf, Buffer.from(ab, 0));
+}
+
+{
+  // If length is not numeric, it defaults to 0.
+  const ab = new ArrayBuffer(10);
+  const buf = Buffer.from(ab, 0, 'fhqwhgads');
+  assert.deepStrictEqual(buf, Buffer.from(ab, 0, 0));
+}
```

Comment:
This is fine, but it may be worthwhile to test additional non-numeric inputs (object, array, NaN, etc).

For instance:
* `Buffer.from(ab, NaN)`, `offset` defaults to `0`
* `Buffer.from(ab, Infinity)` throws
* `Buffer.from(ab, {})` `offset` defaults to `0`
* `Buffer.from(ab, [])` `offset` defaults to `0`
* `Buffer.from(ab, [1])` ``offset is `1`

etc...